core-web/libs/sdk/angular/src/lib/layout/container/container.component.html

```diff
@@ -0,0 +1,35 @@
+@if (isInsideEditor()) {
+<div
+    [attr.data-dot-accept-types]="dotContainer()?.acceptTypes"
+    [attr.data-dot-identifier]="dotContainer()?.identifier"
+    [attr.data-max-contentlets]="dotContainer()?.maxContentlets"
+    [attr.data-dot-uuid]="dotContainer()?.uuid"
+    [style]="contentlets().length ? null : emptyContainerStyles"
+    data-testid="dot-container"
+    data-dot-object="container">
```

Comment:
do we need this extra div? What about binding to the host

Reply:
yeah, binding host for the win!

Reply:
Added!